src/uwtools/drivers/forecast.py

```diff
@@ -150,13 +81,14 @@ def run(self, cycle: datetime) -> bool:
         """
         Runs FV3 either locally or via a batch-script submission.
 
+        :param cycle: the date and start time for the forecast
         :return: Did the FV3 run exit with success status?
         """
         # Prepare directories.
-        run_directory = self._config["run_dir"]
+        run_directory = Path(self._config["run_dir"])
         self.create_directory_structure(run_directory, "delete")
 
-        self._prepare_config_files(Path(run_directory))
+        self._prepare_config_files(cycle, Path(run_directory))
```

Comment:
I see, I think: "config files" in your comment referred to the _component_ config files. We might need some better nomenclature conventions here to differentiate. UW does a LOT with "config files", generally referring to its inputs, rather than its outputs. It might be helpful to have a different term we can use for the component driver configs. I don't have an idea at the moment, but a good name could be helpful.

Reply:
I've made a change here to `_prepare_input_parm`, but we can certainly keep iterating. The [docs for FV3](https://ufs-weather-model.readthedocs.io/en/ufs-v1.0.0/InputsOutputs.html#inputs-and-outputs) call them "input files", but that seemed too broad as input data files are handled somewhere else in our code.

Reply:
Could we consider not propagating the industry-NON-standard term `parm` here? `param` would be better IMO.